CHANGELOG.md

```diff
@@ -4,10 +4,14 @@ See below for Changelog examples.
 
 ## Unreleased
 
-🔧 Fixes:
+🆕 New features:
 
-  - Option Select now uses camelCase instead of snake_case to match govuk-frontend [PR #186](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/186)
+  - New component: Digital Marketplace Keyword search component
```

Comment:
Can we call this 'search box' or just 'search'? I've always found the old 'keyword search' name confusing.

Reply:
Seconded!